pysass.cpp

```diff
@@ -414,13 +555,14 @@ PySass_compile_string(PyObject *self, PyObject *args) {
     Sass_Output_Style output_style;
     int source_comments, error_status, precision, indented;
     PyObject *custom_functions;
+    PyObject *custom_importers;
     PyObject *result;
-
+    
```

Comment:
introduced some redspace here.  I should really add http://pre-commit.com to this project at some point :)